gallery/src/styles/NxTile/NxTileAccordionExample.tsx

```diff
@@ -0,0 +1,111 @@
+            </p>
+          </NxStatefulAccordion>
+          <NxStatefulAccordion>
+            <NxAccordion.Header>
+              <div className="nx-accordion__header-title">Drumstick</div>
+            </NxAccordion.Header>
+            <p className="nx-p">
+              Drumstick ground round hamburger, salami turkey buffalo pork loin tri-tip. Doner kevin pancetta
+              kielbasa ham porchetta strip steak tail buffalo flank pork tri-tip chislic meatloaf. Ribeye tri-tip
+              andouille tail. Pig boudin sausage bacon tail, ham pork loin shank short ribs corned beef capicola
+              brisket bresaola kevin. Chislic strip steak tri-tip, prosciutto burgdoggen short ribs jerky ground
+              round ribeye tongue cow andouille pastrami pig. Chuck andouille sausage cow shankle alcatra tail rump
+              brisket burgdoggen prosciutto.
+            </p>
+          </NxStatefulAccordion>
+          <NxStatefulAccordion>
+            <NxAccordion.Header>
+              <div className="nx-accordion__header-title">Prosciutto</div>
+            </NxAccordion.Header>
+            <p className="nx-p">
+              Prosciutto chislic tail beef sausage short loin spare ribs leberkas fatback landjaeger t-bone bacon
+              andouille venison. Tri-tip salami doner, fatback turkey chicken cow. Cow ball tip rump cupim, burgdoggen
+              jerky bresaola chislic andouille kevin spare ribs. Chuck shoulder bacon doner beef. Biltong alcatra beef
+              ribs salami buffalo ribeye shank sirloin jowl.
+            </p>
+          </NxStatefulAccordion>
+          <NxStatefulAccordion>
+            <NxAccordion.Header>
+              <div className="nx-accordion__header-title">Short Loin</div>
+            </NxAccordion.Header>
+            <p className="nx-p">
+              Short loin bresaola turducken corned beef, chicken rump ground round cupim. Burgdoggen swine picanha tail
+              rump. Meatball pastrami short loin, chicken tail beef ribs turkey doner ham fatback spare ribs pork loin
+              pork belly jowl alcatra. Cow tenderloin pork doner sirloin spare ribs fatback corned beef salami. Short
+              loin t-bone biltong, cupim hamburger kevin jerky beef ribs leberkas corned beef jowl brisket spare ribs
+              capicola pork loin. Shoulder porchetta salami tri-tip burgdoggen, kevin shankle meatball. Ground round
+              capicola beef ribs, cow venison kevin landjaeger strip steak.
+            </p>
+          </NxStatefulAccordion>
+        </div>
+      </section>
+      <section className="nx-tile">
+        <header className="nx-tile-header">
+          <div className="nx-tile-header__title">
+            <h2 className="nx-h2">Tile with normal content and an accordion</h2>
+          </div>
+        </header>
+        <div className="nx-tile-content">
+          <p className="nx-p">
+            Other content may optionally be present outside of the accordion container
```

Comment:
```suggestion
            An <code className="nx-code">nx-tile</code> may have multiple <code className="nx-code">nx-tile-content</code> containers so an accordion can appear within a tile with "normal" content.
```

Reply:
> ```
> -             Other content may optionally be present outside of the accordion container
> +             An <code className="nx-code">nx-tile</code> may have multiple <code className="nx-code">nx-tile-content</code> containers so an accordion can appear within a tile with "normal" content.
> ```

Accepted with minor adjustments in f62a04a6